lib/experimental/Navigation/Dropdown/index.tsx

```diff
@@ -79,9 +98,7 @@ export function Dropdown({ items, children }: DropdownProps) {
               key={index}
               item={{
                 ...item,
-                onClick: () => {
-                  item.onClick()
-                },
+                onClick: item.onClick ? () => item.onClick?.() : undefined,
```

Comment:
This looks redundant, you are checking twice for item.onClick

Reply:
```suggestion
                onClick: () => item.onClick?.()
```

Reply:
Thanks! Can't it be just `onClick: item.onClick`, as it is not defined, it's going to be `undefined` anyway? Or am I missing something?

Reply:
Yep, that should work too

Reply:
What wouldn't work is the original because if onClick is undefined it will break